tdms/include/field.h

```diff
@@ -2,14 +2,44 @@
 #include "simulation_parameters.h"
 
 
+/**
+ * A generic grid entity. For example:
```

Comment:
I'm quite confused by this. Is it a 3D grid? If so shouldn't the coordinates be three-tuples, not two-tuples?

Reply:
they are 3 tuples(?!)

Reply:
As in the coordinates should be e.g. (2, 1, 0), not (2, 1)

Reply:
ohh in the ascii – excellent point 🤦🏼